cvmfs/libcvmfs_int.cc

```diff
@@ -721,17 +721,17 @@ int LibContext::Close(int fd) {
 
 int LibContext::Remount() {
   LogCvmfs(kLogCvmfs, kLogDebug, "remounting root catalog");
-  catalog::LoadError retval =
-    mount_point_->catalog_mgr()->Remount(true /* dry_run */);
+  const catalog::LoadReturn retval =
+                                   mount_point_->catalog_mgr()->RemountDryrun();
+
   switch (retval) {
     case catalog::kLoadUp2Date:
       LogCvmfs(kLogCvmfs, kLogDebug, "catalog up to date");
       return 0;
 
     case catalog::kLoadNew:
-      retval = mount_point_->catalog_mgr()->Remount(false /* dry_run */);
-      if (retval != catalog::kLoadNew)
-        return -1;
```

Comment:
i can add it back.
i dont know why I removed it. maybe by accident (took me quite a while to get the `remount` correctly working). or maybe i thought that because of the `remountDryrun` that has to be successful to go in this branch of the switch statement